resources/views/components/header.blade.php

```diff
@@ -0,0 +1,18 @@
+<!-- /resources/views/components/header.blade.php -->
```

Comment:
Why did you pull this out into a component (vs ponerlo en el layout)?

Reply:
moved it!